client/context/context.go

```diff
@@ -113,3 +116,15 @@ func (ctx CLIContext) WithUseLedger(useLedger bool) CLIContext {
 	ctx.UseLedger = useLedger
 	return ctx
 }
+
+// WithCert - return a copy of the context with an updated Cert
```

Comment:
Use [Tendermint Lint](https://github.com/tendermint/lint) to avoid verbose comments. You won't need to add the function name at the beginning of the comment with it